core/src/main/java/org/elasticsearch/index/query/DisMaxQueryBuilder.java

```diff
@@ -114,11 +115,13 @@ public float tieBreaker() {
     protected void doXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject(NAME);
         builder.field(TIE_BREAKER_FIELD.getPreferredName(), tieBreaker);
+        if (queries.isEmpty() == false) {
```

Comment:
indentation looks off here. btw shall we output no array or an empty array? :) what were we doing before? not sure this change is needed.